### ISSUE TITLE
Mysql install freeze

### ISSUE THREAD

**Issue by pmargreff** [nkratzke/EasyMySQL#4]
When the docker started install `mysql`, the install don't continue after root pass input: 

```
While not mandatory, it is highly recommended that you set a password for the
MySQL administrative "root" user.

If this field is left blank, the password will not be changed.

New password for the MySQL "root" user: 
```

Someone could help with that?

**Comment by nkratzke** [comment]
Should be fixed with commit a25f80e2067f30148a86ddfe58682f57366eb1f0.

**Comment by pmargreff** [comment]
Works well man, thanks.
